src/erldns_packet_cache.erl

```diff
@@ -75,14 +77,16 @@ get(Question, _Host) ->
 %% @doc Put the response in the cache for the given question.
 -spec put(dns:question(), dns:message()) -> ok.
 put(Question, Response) ->
-  case ?ENABLED of
-    true ->
+  ?IF(
```

Comment:
The reason for this is that when you guys are using macros to enable / disable things at compile time it makes a statement like
```
case ?ENABLED of
    true ->
        foo;
    false ->
        bar
end.

into:
case true of
    true ->
        foo;
    false ->
        bar
end.
```

Dialyzer doesn't quite understand this. This is basically an implementation of if-than-else with the parse transform implemented elsewhere.